sync.go

```diff
@@ -75,14 +75,17 @@ func (s *Syncer) Start(ctx context.Context, socket commSocket) {
 		close(initChan)
 
 		defer fmt.Println("clipboard listener stopped")
-		clipboardChanges := clipboard.Watch(s.ctx, clipboard.FmtText)
-		for newContent := range clipboardChanges {
-			s.mtx.Lock()
+
+		// Keep checking for clipboard changes on the host machine
+		for {
```

Comment:
This doesn't seem to have any exit condition. Previously, when we closed the clipboard, the watcher channel would be closed as well, therefore exiting the loop.

Reply:
True , mb. Also  i noticed that i want two things going at the same time and is making my brain to work. If i wanted to notify the client that it may need to wait should i ask it to wait a set amount of time then ? Unless it gets updated then it just restarts the loop.